runtime/libpgmath/lib/CMakeLists.txt

```diff
@@ -235,23 +235,19 @@ endif()
 # Create libmath static library
 get_property(TARGET_OBJECTS GLOBAL PROPERTY "TARGET_OBJECTS")
 
-add_library(${LIBPGMATH_LIBRARY_NAME}_static STATIC ${TARGET_OBJECTS})
-add_library(${LIBPGMATH_LIBRARY_NAME} SHARED ${TARGET_OBJECTS})
-target_link_libraries(${LIBPGMATH_LIBRARY_NAME} INTERFACE ${LIBPGMATH_LIBRARY_NAME}_static)
-if(${LIBPGMATH_SYSTEM_NAME} MATCHES "Windows")
-    set_target_properties(${LIBPGMATH_LIBRARY_NAME}_static PROPERTIES OUTPUT_NAME lib${LIBPGMATH_LIBRARY_NAME})
-    set_target_properties(${LIBPGMATH_LIBRARY_NAME} PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS TRUE)
-if(${CMAKE_GENERATOR} MATCHES "MSYS Makefiles")
-add_custom_command (TARGET ${LIBPGMATH_LIBRARY_NAME} PRE_LINK
-    COMMAND  sed 's/ /\\n/g' CMakeFiles/pgmath.dir/objects1.rsp > objects1.txt
-    COMMAND  ${CMAKE_COMMAND} -E __create_def libpgmath.def objects1.txt
-)
-endif()
+add_library(pgmath_shared SHARED ${TARGET_OBJECTS})
+add_library(pgmath_static STATIC ${TARGET_OBJECTS})
 
-else()
-    set_target_properties(${LIBPGMATH_LIBRARY_NAME}_static PROPERTIES OUTPUT_NAME ${LIBPGMATH_LIBRARY_NAME})
-endif()
-install(TARGETS ${LIBPGMATH_LIBRARY_NAME}
-  LIBRARY DESTINATION lib)
-install(TARGETS ${LIBPGMATH_LIBRARY_NAME}_static
-  ARCHIVE DESTINATION lib)
+target_link_libraries(pgmath_shared INTERFACE pgmath_static)
+set_target_properties(pgmath_shared PROPERTIES OUTPUT_NAME pgmath WINDOWS_EXPORT_ALL_SYMBOLS TRUE)
```

Comment:
doesn't affect other platforms. Also, the library name is independent of the filename for cmake, so we don't need to use the library name variable.